helpers.cpp

```diff
@@ -13,7 +13,25 @@ void grayscale(const int &height, const int &width, RGBTRIPLE **image) {
 
 // Definition for Reflect filter (Horizontally)
 void reflect(const int &height, const int &width, RGBTRIPLE **image) {
-    // code to be written
+   for (int i {0}; i < height; i++) {
+        for (int j {0}; j < width / 2; j++) {
```

Comment:
```suggestion
   for (int i {0}; i < height; ++i) {
        for (int j {0}; j < width / 2; ++j) {
```
Please prefix instead of postfix.

Reply:
yes , I have updated that one @Himanshu40